packages/jss-starter-kit/src/index.js

```diff
@@ -10,7 +10,16 @@ jss.setup(preset())
 
 export {jss, preset}
 
-export * as reactJss from 'react-jss'
+export {SheetsRegistry, SheetsManager, createGenerateId} from 'jss'
+
+export {
```

Comment:
But then you couldn’t import them directly which isn’t user-friendly in my opinion

Reply:
Its not too user friendly, but on the other hand, those exports might have potential collision now and because we need to maintain the named exports here ass well.

Reply:
I don't think there is a way to have every member exported right now. Rollup doesn't bundle something like `export * from 'react-jss` correctly I believe.

@TrySound do we need to enable something to support this syntax?

Reply:
Rollup is able yo handle export * syntax just fine. It expands it into named imports internally.

`Export * as ns` is not supported since its not a part of spec yet.

Reply:
Ah, I see. It does not expand this because it doesnt know about external dependency. Who resolves this incorrectly? Webpack?

Reply:
Figured it out. The problem was that `react-jss` wasn't declared a dependency of `jss-starter-kit`.